nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -67,10 +69,12 @@ class LhotseDataLoadingConfig:
     quadratic_duration: float | None = None
     #   c. Lhotse bucketing.
     use_bucketing: bool = False
+    bucket_batch_size: list[int] | None = None
     num_buckets: int = 30
     num_cuts_for_bins_estimate: int = 10000
-    bucket_duration_bins: list[float] | None = None
+    bucket_duration_bins: Any = None  # list[float] | list[list[float]] | None = None
     bucket_buffer_size: int = 10000
+    concurrent_bucketing: bool = True  # fetches data in a background thread
```

Comment:
Do you have any concern that this might increase memory unreasonably?

e.g., is there a limit to how much the background thread will load data? e.g., in pytorch there is a prefetch_factor option to prevent unbounded memory loading if the data loader is faster than the consumer.